p2p/vpacket.go

```diff
@@ -1,17 +1,19 @@
 /**
-*  @file
+*  Package p2p
 *  @copyright defined in go-seele/LICENSE
  */
 package p2p
 
-import "encoding/binary"
+import  (
+    "encoding/binary"
+)
 
 const (
     //VPacketHeadLen head length in net-package
```

Comment:
dataLen don't need send to peer, because in the receiver end it can be calculated by udp package size and VPacketHeadLen.

Reply:
sure, I already reverted to original version.